src/components/Sections/SectionMarkdown.js

```diff
@@ -76,7 +77,11 @@ export default class SectionMarkdown extends Component {
       }
       case 'img': {
         const srcArr = props.src.split('=size=');
-        props.src = srcArr[0];
+        if (srcArr[0].startsWith(MARKDOWN_IMAGES_PATH)) {
+          props.src = `${markdownArtifactsServerAddress}${srcArr[0]}`;
```

Comment:
No need to add `/` if not exists?

Reply:
the path in `MARKDOWN_IMAGES_PATH` already have `/`